app/dto/model/customer.py

```diff
@@ -23,3 +23,20 @@ class CustomerDTOs(BaseModel):
 
     class Config:
         orm_mode = True
+
+
+class SignupDTO(BaseModel):
```

Comment:
in the dto folder create a package call controller-> auth.py file and move all dto needed in auth controller to the auth.py file